src/api/routes/faucets.ts

```diff
@@ -75,7 +75,7 @@ export function createFaucetRouter(db: DataStore): RouterWithAsync {
       // Only based on address for now, but we're keeping the IP in case
       // we want to escalate and implement a per IP policy
       const now = Date.now();
-      const window = 5 * 60 * 1000; // 5 minutes
+      const window = 2 * 24 * 60 * 60 * 1000; // 2 days
```

Comment:
I think people will run into issues with a 2 day throttle, but not sure of a good alternative.

Reply:
not too hard to "circumvent" by using multiple addresses...

Reply:
Updated to 1 hour

Reply:
As @psq mentioned, if for some reason (?) this backoff is an issue, users can just use a different address.

Reply:
Opening https://github.com/blockstack/stacks-blockchain-api/issues/247, for the sustainable approach